winter/routing/routing.py

```diff
@@ -13,39 +13,51 @@
 class Route:
     url_path: str
     http_method: str = None
-    produces: Tuple[MediaType] = None  # It's used for swagger only at the moment, but will be used in routing later
+    produces: Tuple[MediaType] = ()  # It's used for swagger only at the moment, but will be used in routing later
+    consumes: Tuple[MediaType] = ()  # It's used for swagger only at the moment, but will be used in routing later
 
 
-def route(url_path: str, http_method: Optional[str] = None, produces: Optional[Tuple[MediaType]] = None):
+def route(
+        url_path: str,
+        http_method: Optional[str] = None,
+        produces: Optional[Tuple[MediaType]] = (),
+        consumes: Optional[Tuple[MediaType]] = (),
```

Comment:
Why empty tuple instead of None? I think that `None` means `not specified` while empty tuple tells `nothing is produced. What do you think? Also, if empty tuple is ok, then remove `Optional`.